src/components/NavButton/style.css

```diff
@@ -47,16 +58,28 @@
     color: var(--dark-theme-secondary-button-color);
 }
 
-.nav-button.secondary-button:global(.ant-btn:hover),
-.nav-button.secondary-button:global(.ant-btn:focus) {
+.nav-button.secondary-button:global(.ant-btn:hover) {
     color: var(--dark-theme-secondary-button-hover-color);
     background-color: var(--dark-theme-secondary-button-hover-bg);
     border-color: var(--dark-theme-secondary-button-hover-border);
 }
 
+.nav-button.secondary-button:global(.ant-btn:focus) {
+    color: var(--dark-theme-secondary-button-focus-color);
+    background-color: var(--dark-theme-secondary-button-focus-bg);
+    border-color: var(--dark-theme-secondary-button-focus-border);
+    outline: 1px solid var(--dark-theme-secondary-button-focus-outline);
+    outline-offset: 2px;
+}
+
 .nav-button.secondary-button:global(.ant-btn).disabled {
     color: var(--dark-theme-secondary-button-disabled-color);
     background-color: var(--dark-theme-secondary-button-disabled-bg);
     border-color: var(--dark-theme-secondary-button-disabled-border);
     cursor: not-allowed;
+}
+
+.nav-button.dropdown-item-button:global(.ant-btn) {
+    background-color: inherit;
+    border: inherit;
```

Comment:
In some cases we were using `<a>` to open modals but for accessibility those should be a buttons, even if its easier to style when they are all the same element type. I also felt the styling for all the differenet elements that can be dropdown menu items should be in the `CustomDropdown` stylesheet.

Using `<NavButton />`, making the new `DropdownItem` button class, and using these two inherit rules seemed to be the most concise way to manage selector specificity and keep the styling where I wanted it. Open to suggestions.